part4/README.md

```diff
@@ -102,63 +99,64 @@ export interface Todo {
 }
 ```
 
-If we now navigate to "http://localhost:5000/todo" we should see the data from the database displayed in the browser.
-If you encountered problems, do not hesitate to ask for help or checkout the solution in the part4-solution-example branch.
+Etter endringene må du bygge containeren med programmet på nytt.
 
-### Documentation
+Nå kan vi gå til http://localhost:5000/todo og se at siden _prøver_ å hente data fra databasen. Vi skal hjelpe den.
+
+Hvis du har problemer med å få den til å kjøre er det bare å spørre om hjelp eller se løsningsforslaget på `part4-solution-example`-branchen.
+
+### Dokumentasjon
 
 SvelteKit routing (+server) https://kit.svelte.dev/docs/routing#server
 
 SvelteKit server side modules: https://kit.svelte.dev/docs/server-only-modules
 
-## 4.6 Creating a network for the DB and application to communicate
+## 4.6 Lage et nettverk mellom containerne
 
 ```bash
-podman network create todo
+podman network create todonet
 ```
 
-This will create a network called todo
+Dette bil opprette et nettverk kalt `todonet`.
 
-### Running the DB and the application in the same network
+### Kjøre containerne på samme nett
 
-The DB:
+Vi starter med å spinne opp databbasen:
 
 ```bash
-podman run -d --network todo postgres
+podman run -d --network todonet <tagname til db>
 ```
 
-Note the absence of the -p flag. We do not need to expose the port of the container to the host as the FE will be communicating with the BE through a network.
+Observer at vi ikke bruker `-p`-flagget. Vi trenger ikke eksponere porten til containeren videre til verten ettersom programmet vil kommunisere med databasen over nettverket vi har laget.
 
-And now join the rest of the application on the same network (remember to build the image if you have done changes since last build!):
+Så spinner vi opp resten av programmet:
 
 ```bash
-podman run -d --network todo -p 5000:5000 nameOfImage:latest
+podman run -d --network todo -p 5000:5000 <tagname til app>:latest
 ```
 
-To be able to reach the application, we must still expose port 5000 on the host.
+Her må vi bruke `-p`-flagget for å eksponere port 5000 til verten slik at vi kan nå programmet fra nettleseren.
 
 ### Problem: The FE cannot reach the DB
```

Comment:
@SteinarVrenne Er dette alltid tilfelle? I så fall kan jeg skrive det litt om så deltagerne slipper å gjøre feil først.

Reply:
Yes, port 5000 må alltid eksponeres for å nås i nettleser :)

Reply:
Mente egentlig linje 140. Vil punkt 4.6 alltid feile før man har utført det under? I så fall kan vi vurdere om vi kan skrive det om så man ikke blir ferdig med første del av 4.6 uten at det fungerer. Kan diskutere det i morgen, rekker ikke før denne workshoppen uansett 😊